pkgs/by-name/ar/argc/package.nix

```
@@ -37,6 +38,13 @@ rustPlatform.buildRustPackage rec {
 
   disallowedReferences = lib.optional (!canExecuteHost) buildPackages.argc;
 
+  env = {
+    LANG = "C.UTF-8";
+    LOCALE_ARCHIVE = lib.optionalString (
+      glibcLocales != null
+    ) "${glibcLocales}/lib/locale/locale-archive";
+  };
```

```suggestion
    LANG = "C.UTF-8";
  } // lib.optionalAttrs (glibcLocales != null) {
    LOCALE_ARCHIVE = "${glibcLocales}/lib/locale/locale-archive";
  };
```
a little bit cleaner

---

Yeah, good point. Implemented in ac93a0467d7a9b85d62fd10e838ad362ec458c83.